lib/bundler_helper.rb

```diff
@@ -19,9 +19,11 @@ def self.require_groups(*groups)
             exit 1
           end
         end
+        Bundler.ui = Bundler::UI::Silent.new
         BundlerExt.system_require(File.expand_path('../Gemfile.in', __dir__), *groups)
       else
         require 'bundler' unless defined?(Bundler)
+        Bundler.ui = Bundler::UI::Silent.new
```

Comment:
In production we should have `Gemfile.in` and use `bundler_ext` and this would be the development case. I think we should leave it out here.
```suggestion
```